acj/__init__.py

```diff
@@ -4,9 +4,10 @@
 from flask import Flask, redirect, session as sess, abort, jsonify
 from flask_login import current_user
 from sqlalchemy.orm import joinedload
+from celery import Celery
 
 from .authorization import define_authorization
-from .core import login_manager, bouncer, db, cas
+from .core import login_manager, bouncer, db, cas, celery
```

Comment:
Yes, I'm writing over the value on app init

Celery unfortunately doesn't seem to have the `celery.init_app()` function that a lot of the other flask libraries have